### PR TITLE
Update CNI Plugins to v1.1.1

### PR DESCRIPTION
**Pull request by orsenthil**
**What type of PR is this?**

Improvement - Update CNI Plugins to v1.1.1

**Which issue does this PR fix**:

Upgrades the CNI plugins used by vpc-cni

**What does this PR do / Why do we need it**:

General Improvement.

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

N/A

**Will this PR introduce any new dependencies?**:

N/A

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

No

**Does this PR introduce any user-facing change?**:

No

```release-note
Includes v1.1.1 of CNI Plugins `loopback` `portmap` `bandwidth` and `host-local`

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -221,7 +221,7 @@ multi-arch-cni-init-build-push:     ## Build VPC CNI plugin Init container image
 		--push \
 		.
 # Fetch the CNI plugins
-plugins: FETCH_VERSION=0.9.0
+plugins: FETCH_VERSION=1.1.1
```
Comment (jayanthvn) [inline Makefile:224]: Maybe even here we can update - https://github.com/aws/amazon-vpc-cni-k8s/blob/master/go.mod#L8

Comment (orsenthil) [reply Makefile:224]: It is a good idea. This is going to bring few changes.

```diff --git a/go.mod b/go.mod
index 21ab04f7..36adb6b1 100644
--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.18
 require (
        github.com/aws/aws-sdk-go v1.43.29
        github.com/containernetworking/cni v0.8.1
-       github.com/containernetworking/plugins v0.9.0
+       github.com/containernetworking/plugins v1.1.1
        github.com/coreos/go-iptables v0.4.5
        github.com/golang/mock v1.4.1
        github.com/golang/protobuf v1.4.3
```

And

```
$ go mod tidy
go: downloading github.com/containernetworking/plugins v1.1.1
go: downloading github.com/onsi/gomega v1.15.0
go: downloading github.com/containernetworking/cni v1.0.1
go: downloading github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
go: downloading github.com/safchain/ethtool v0.0.0-20210803160452-9aa261dae9b1
go: downloading github.com/vishvananda/netns v0.0.0-20210104183010-2eb08e3e575f
go: finding module for package github.com/containernetworking/cni/pkg/types/current
go: downloading github.com/containernetworking/cni v1.1.1
github.com/aws/amazon-vpc-cni-k8s/cmd/egress-v4-cni-plugin imports
	github.com/containernetworking/cni/pkg/types/current: module github.com/containernetworking/cni@latest found (v1.1.1), but does not contain package github.com/containernetworking/cni/pkg/types/current
```

the resolution seems like places we are using `types/current` (around 5 files) It should be updated.
Ref https://www.cni.dev/docs/spec-upgrades/#libcni-changes-in-cni-v10

Comment (jayanthvn) [reply Makefile:224]: Yeah maybe binary makes sense for now considering the criticality? @achevuru or @M00nF1sh any thoughts?

Comment (orsenthil) [reply Makefile:224]: tried to upgrade the library. We shall bring it separately. as removing of `Version` field fails `make` binaries.
We also have find a reliable way to different v4 and v6 addresses without that field.

```
# Updating to latest version.

$ make
go build  -buildmode=pie -ldflags '-s -w -X pkg/version/info.Version=v1.6.4-rc1-379-gc17158ef-dirty -X pkg/awsutils/awssession.version=v1.6.4-rc1-379-gc17158ef-dirty' -o aws-k8s-agent     ./cmd/aws-k8s-agent
go build  -buildmode=pie -ldflags '-s -w -X pkg/version/info.Version=v1.6.4-rc1-379-gc17158ef-dirty -X pkg/awsutils/awssession.version=v1.6.4-rc1-379-gc17158ef-dirty' -o aws-cni           ./cmd/routed-eni-cni-plugin
# github.com/aws/amazon-vpc-cni-k8s/cmd/routed-eni-cni-plugin
cmd/routed-eni-cni-plugin/cni.go:268:4: unknown field 'Version' in struct literal of type types100.IPConfig
make: *** [build-linux] Error 2

# Removing Version field as it was redundant

$ make
go build  -buildmode=pie -ldflags '-s -w -X pkg/version/info.Version=v1.6.4-rc1-379-gc17158ef-dirty -X pkg/awsutils/awssession.version=v1.6.4-rc1-379-gc17158ef-dirty' -o aws-k8s-agent     ./cmd/aws-k8s-agent
go build  -buildmode=pie -ldflags '-s -w -X pkg/version/info.Version=v1.6.4-rc1-379-gc17158ef-dirty -X pkg/awsutils/awssession.version=v1.6.4-rc1-379-gc17158ef-dirty' -o aws-cni           ./cmd/routed-eni-cni-plugin
go build  -buildmode=pie -ldflags '-s -w -X pkg/version/info.Version=v1.6.4-rc1-379-gc17158ef-dirty -X pkg/awsutils/awssession.version=v1.6.4-rc1-379-gc17158ef-dirty' -o grpc-health-probe ./cmd/grpc-health-probe
go build  -buildmode=pie -ldflags '-s -w -X pkg/version/info.Version=v1.6.4-rc1-379-gc17158ef-dirty -X pkg/awsutils/awssession.version=v1.6.4-rc1-379-gc17158ef-dirty' -o egress-v4-cni     ./cmd/egress-v4-cni-plugin
# github.com/aws/amazon-vpc-cni-k8s/cmd/egress-v4-cni-plugin
cmd/egress-v4-cni-plugin/cni.go:124:57: not enough arguments in call to ip.SetupVeth
	have (string, int, ns.NetNS)
	want (string, int, string, ns.NetNS)
cmd/egress-v4-cni-plugin/cni.go:166:11: ipc.Version undefined (type *types100.IPConfig has no field or method Version)
cmd/egress-v4-cni-plugin/cni.go:325:11: ipc.Version undefined (type *types100.IPConfig has no field or method Version)
make: *** [build-linux] Error 2


#  Without Version field we will have to find a reliable to to different IPV4 and IPV6 in the cmds we use.
[senthilx@88665a371033:~]$
```

**Comment by orsenthil** [comment]
The integration test was successful - https://github.com/aws/amazon-vpc-cni-k8s/runs/6716436966?check_suite_focus=true

**Review by jayanthvn — APPROVED** [review]

**Pull request merged**